### ISSUE TITLE
urllib.error.HTTPError: HTTP Error 404: Not Found

### ISSUE THREAD

**Issue by abutremutante** [GeneralMills/pytrends#63]
Hello everyone,

Had this problem which the traceback you can see below. I've seen it is the same problem described here: https://github.com/GeneralMills/pytrends/issues/30

I've done what was suggested there, to update pytrends through pip, however it didn't solve the problem, I keep having it. I'm using python 3.5.0 and the guy in the link above had 3.5.1. Would that be the problem?

TRACEBACK:
    /Library/Frameworks/Python.framework/Versions/3.5/bin/python3.5 "/Users/santanna_santanna/PycharmProjects/Predictive Models/APIGtrends.py"
    Traceback (most recent call last):
    File "/Users/santanna_santanna/PycharmProjects/Predictive Models/APIGtrends.py", line 19, in <module>
    connector = pyGTrends(google_username, google_password)
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pytrends/pyGTrends.py", line 41, in __init__
    self.fake_ua = UserAgent()
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/fake_useragent/fake.py", line 10, in __init__
    self.data = load_cached()
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/fake_useragent/utils.py", line 140, in load_cached
    update()
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/fake_useragent/utils.py", line 135, in update
    write(load())
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/fake_useragent/utils.py", line 92, in load
    browsers_dict[browser_key] = get_browser_versions(browser)
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/fake_useragent/utils.py", line 53, in get_browser_versions
    html = get(settings.BROWSER_BASE_PAGE, browser)
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/fake_useragent/utils.py", line 20, in get
    return urlopen(url).read()
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/urllib/request.py", line 162, in urlopen
    return opener.open(url, data, timeout)
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/urllib/request.py", line 471, in open
    response = meth(req, response)
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/urllib/request.py", line 581, in http_response
    'http', request, response, code, msg, hdrs)
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/urllib/request.py", line 509, in error
    return self._call_chain(*args)
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/urllib/request.py", line 443, in _call_chain
    result = func(*args)
    File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/urllib/request.py", line 589, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

**Comment by dreyco676** [comment]
Umm try `pip install pytrends --upgrade` you have an old version because we don't use fake agent anymore.

**Comment by dreyco676** [comment]
Reopen if persists after updating with `--upgrade`

**Comment by abutremutante** [comment]
Didn't work but it was great to know that the problem was with the pytrends version, it made me find my way to fix it. I will write down my solution, it may be other's problem someday:

I'm using PyCharm. Noticed that the version of pytrends that PyCharm was using was not the latest (3.2.0). So I manually specified the version of pytrends I wanted to install and actually it installed.

However, apparently you have changed methods names as well. 

I was importing it using the following code:
from pytrends.pyGTrends import pyGTrends
import time
from random import randint

And I was getting the following Traceback

Traceback (most recent call last):
  File "/Users/santanna_santanna/PycharmProjects/file.py", line 10, in <module>
    from pytrends.pyGTrends import pyGTrends
ImportError: No module named 'pytrends.pyGTrends'

That was the moment I noticed you changed your code substantially. Looks good!
Now I'm adapting my code according to your new example here: https://github.com/GeneralMills/pytrends/blob/master/examples/example.py

Thanks for the tip!
